bdb/info.c

```diff
@@ -689,7 +689,7 @@ void fill_dbinfo(void *p_response, bdb_state_type *bdb_state)
             nodeinfos[i]->room =
                 bdb_state->callback->getroom_rtn(bdb_state, nodes[j].host);
             nodeinfos[i]->name = strdup(nodes[j].host);
-            if (strcmp(bdb_state->repinfo->master_host, nodes[i].host) == 0) {
+            if (strcmp(bdb_state->repinfo->master_host, nodes[j].host) == 0) {
                 master->number = 0; /* will not be used by client */
```

Comment:
can we have master->number = j
This will make it work with bbgithub cdb2api too without any change on its end.

Reply:
nodeinfos[i]->number = j 
too @ line 685